CvGameCoreDLL_Expansion2/CvUnit.cpp

```diff
@@ -31300,100 +31308,266 @@ int CvUnit::AI_promotionValue(PromotionTypes ePromotion)
 		}
 	}
 
+	*/
+
+			// This part is hard to balance		
```

Comment:
Because it searches through all the combat classes and counts each one. So formation gets hit once when UNITCOMBAT_MOUNTED is hit then air supremacy twice with bombers and fighters. But then encirclement and breacher get hit 4 times with navalmelee, navalranged, sub and carrier. So I can't figure out how to balance this without just tailoring it for a specific promotion.
I agree the *0.8 and *1.6 so probably be changed to *1 and *2 here.